docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -169,10 +136,10 @@ Save this as `teleport-pagerduty.service`.
   Your browser does not support the video tag.
 </video>
 
-## On-call Auto Approval
+## On-call auto-approval
 
 The PagerDuty plugin has an option to auto-approve access requests. This
-feature will map external SSO identity to PagerDuty on call email addresses.
+feature will map external SSO identity to PagerDuty on-call email addresses.
```

Comment:
"will map external SSO identity to PagerDuty on-call email addresses" should probably be "will map external an SSO identity to a PagerDuty on-call email address"